code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```diff
@@ -693,7 +690,8 @@ func (fsh *StorageHandler) UpdateObjectAttributes(ctx context.Context, r *http.R
 	defer mutex.Unlock()
 
 	var ref *reference.Ref
-	ref, err = reference.GetReferenceFromLookupHash(ctx, alloc.ID, pathHash)
+	// Update GetReferenceFromLookupHash to GetOnlyReferencePathFromLookupHash
+	ref, err = reference.GetOnlyReferencePathFromLookupHash(ctx, alloc.ID, pathHash)
```

Comment:
Would `GetReferencePathFromLookupHash` be better than `GetOnlyReferencePathFromLookupHash`?